README.md

```diff
@@ -174,6 +174,10 @@ Time::DATE_FORMATS[:simple] = "%-l:%M%P"
 Time::DATE_FORMATS[:simple_24h] = "%H:%M"
 ```
 
+To format the relative time e.g. when `:type` is set to `time-ago`, `config.useFormat24` is used, too. Depending on the configuration the following keys are used to get for the corresponding formats from the `I18n` configuration: `time.formats.default` or `time.formats.default_24h`.
+
+For the title of the html element either `datetime.formats.default` or `datetime.formats.default_24h` is used depending on the configuration set.
```

Comment:
```suggestion
When `:type` is set to `time-ago`, the format is obtained from the `I18n` [configuration](#configuration).
```

I think this is enough of an explanation for now. I plan to revisit the readme as a whole for the next release.